packages/hub/Dockerfile

```
@@ -8,6 +8,13 @@ COPY dist/sql node_modules/graphile-worker/sql
 # This includes the discord bot image assets that are hosted by the hub
 COPY dist/services packages/hub/services
 
+# Copy Prisma necessities
+COPY dist/@prisma/client node_modules/.prisma/client
+COPY dist/@prisma/client node_modules/@prisma/client
+COPY prisma/schema.prisma node_modules/.prisma/client/schema.prisma
+COPY prisma/schema.prisma node_modules/@prisma/client/schema.prisma
+COPY dist/.prisma/client/*.node node_modules/@prisma/client/
```

Is this a typo? If so we might not need the subsequent COPY instructions for `dist/.prisma/<somefile>`, or for `schema.prisma`? My understanding is `@prisma/client` requires `.prisma/client`, so as long as we copy both of them into `node_modules` we should be ok. This is the code from `@prisma/client/index.js`:

```js
module.exports = {
  ...require('.prisma/client/index'),
}
```

```suggestion
COPY dist/.prisma/client node_modules/.prisma/client
COPY dist/@prisma/client node_modules/@prisma/client
```

---

Also just realised we probably don't need to `yarn add @prisma/client` as a final step, as specified in the readme.